common/src/main/java/com/scottlogic/deg/generator/utils/SetUtils.java

```diff
@@ -40,6 +42,7 @@
         return Collections.unmodifiableSet(Stream.of(e1, e2, e3).collect(Collectors.toSet()));
     }
 
+    @SafeVarargs
```

Comment:
Including this annotation seems rogue. Are you sure that this method is "safe"? If not, I think the annotation should be removed.

Reply:
The varargs in this method do not mutate the array created, or pass a reference to the array out of the method. Therefore, the varargs are safe.